src/components/controls/panelHeader.tsx

```diff
@@ -4,22 +4,39 @@ import React from "react";
 import { HeaderContainer } from "./styles";
 import PanelToggle from "./panelToggle";
 import { AnnotatedTitle } from "./annotatedTitle";
+import { PanelChevron } from "./panelChevron";
 
 type Props = {
   panel: string
   title: string
   tooltip: JSX.Element
   panelIsVisible: boolean
+  hasOptions: boolean
+  optionsAreVisible: boolean
+  setOptionsAreVisible: React.Dispatch<React.SetStateAction<boolean>>
   mobile: boolean
 }
 
-export const PanelHeader = ({ panel, title, tooltip, panelIsVisible, mobile }: Props) => {
+export const PanelHeader = ({ panel, title, tooltip, panelIsVisible, hasOptions, optionsAreVisible, setOptionsAreVisible, mobile }: Props) => {
+
+  let titleContainerProps = {}
+
+  if (hasOptions) {
+    titleContainerProps = {
+      role: "button",
+      onClick: () => setOptionsAreVisible(!optionsAreVisible),
+    }
+  }
+
   return (
     <HeaderContainer>
-      <AnnotatedTitle
-        title={title}
-        tooltip={tooltip}
-        mobile={mobile} />
+      <span {...titleContainerProps}>
+        {hasOptions && <PanelChevron show={optionsAreVisible} />}
```

Comment:
If we do switch the behavior here, I would remove the chevron entirely for mobile display.

Reply:
In this [force-push](https://github.com/nextstrain/auspice/compare/830b73de2d4d03f42b7272e8fd6f83e6616cfed1..6753065b16fafa781fa44424e66852cd56a2f33d), among other smaller changes, I made it so that clicking most of the header changes the toggle.

Notes:

- Clicking the info icon does not change the toggle, since I imagine that can only be unintentional.
- Clicking the chevron is the only thing that changes options visibility, and is not rendered on mobile display.

Reply:
Thanks. Behaviour works well. One minor bug - the "filter data", "color by" headers shouldn't have a "pointer" cursor.

Reply:
Oh right, good catch.

Reply:
Fixed in [force-push](https://github.com/nextstrain/auspice/compare/b25e783edcc63d11798541f2ee207bd8aacb3f06..a8f8b0dad73419da5b42d5edbd979cb6752888d7).

Reply:
After collecting feedback [on Slack](https://bedfordlab.slack.com/archives/C01LCTT7JNN/p1699577373284099?thread_ts=1696895285.086759&cid=C01LCTT7JNN), I've [force-pushed](https://github.com/nextstrain/auspice/compare/a8f8b0dad73419da5b42d5edbd979cb6752888d7..e3eabae6ddec32dbcd68e7159d24efcee5d0b4ec) another change in behavior.

Instead of making most of the click area toggle panel visibility, make it toggle the options visibility.

Reply:
👍